static/app/components/events/interfaces/spans/newTraceDetailsSpanDetails.tsx

```diff
@@ -523,20 +524,20 @@ function NewTraceDetailsSpanDetail(props: SpanDetailProps) {
                 <Row title={key} key={key}>
                   <Fragment>
                     <FileSize bytes={value} />
-                    {value >= 1024 && <span>{` (${maybeStringify(value)} B)`}</span>}
+                    {value >= 1024 && <span>{` (${value} B)`}</span>}
```

Comment:
this is always a number

Reply:
Is this true for all size attributes and all sdk versions? I'm thinking javascript in particular where this assumption may not hold since the data attribute for spans permits free form data so perhaps consider adding it back for safety.